src/nwbinspector/_nwb_inspection.py

```diff
@@ -73,8 +71,8 @@ def inspect_all(
         all available CPUs minus x.
         Set to 1 (also the default) to disable.
     skip_validate : bool, optional
-        Skip the PyNWB validation step. This may be desired for older NWBFiles (< schema version v2.10).
```

Comment:
I think this is ok to keep as a comment here and below. 
From what I understand skipping the validation step for older NWBFiles is a different issue from using an older version of pynwb?

Reply:
It's mostly that I don't want to make assumptions about why a user might want to skip validation - or in particular, give them a ready excuse to do so

Historically the main use is to prevent errors from occurring if there's some unexpected issue using an old PyNWB version on a newer file

But I trust modern PyNWB to either work on previous schema versions or otherwise trust the Inpspector to route an error to the right part of the messages